tests/store/test_sqlalchemy_store.py

```diff
@@ -474,6 +484,7 @@ def test_log_param(self):
         param2 = entities.Param('new param', 'new key')
         self.store.log_param(run.info.run_uuid, param)
         self.store.log_param(run.info.run_uuid, param2)
+        self.store.log_param(run.info.run_uuid, param2)
```

Comment:
Test that we can log the same param twice (we didn't have a test for this before AFAICT)